client/electron/go_helpers.ts

```diff
@@ -34,17 +34,19 @@ import {ChildProcessHelper} from './process';
  */
 export async function checkUDPConnectivity(
   config: string,
+  adapterIp: string = null,
```

Comment:
These only apply to Windows.

Please have separate interfaces for separate platforms that need different parameters.
Have the platform name in the function name (e.g. `checkUdpConnectivityWindows`)

Reply:
Added